CONTRIBUTING.md

```diff
@@ -26,9 +29,63 @@ The Corporate CLA is for those who are contributing as a member of an organizati
 - [CLA for individuals](https://opensource.atlassian.com/individual)
 - [CLA for corporate contributors](https://opensource.atlassian.com/corporate)
 
+## Key Concepts
+
+### CSS-in-JS
+
+Compiled is a CSS-in-JS library: a library that lets React developers write CSS within their React code. Other libraries in this space include styled-components and Emotion. Below are some code examples.
```

Comment:
Compiled should be framework agnostic, at least it's trying to be 🤦‍♂️ 
```suggestion
Compiled is a CSS-in-JS library: a library that lets frontend developers write CSS within their JavaScript code. Other libraries in this space include styled-components and Emotion. Below are some code examples.
```

Reply:
I might be misremembering here, but isn't React the only major framework without an inbuilt CSS-in-JS solution? 😅